dind/README.md

```diff
@@ -0,0 +1,46 @@
+# rke DinD
+
+Docker-in-Docker (DinD) feature is created mainly for testing.
+
+DinD creates docker containers locally which are then used as cluster nodes.
+
+It still requires a cluster.yml with minimal information:
+
+```
+kubernetes_version: "$KUBE_VERSION"
+nodes:
+- address: foo
```

Comment:
Possibly add this needs to be unique even though it's a dummy

Reply:
ok